README.md

```diff
@@ -171,6 +171,18 @@ interval between any two periodic sync events can be greater than the requested
 is up to the browser implemetation. Also, the interval between any two periodic sync events for a
 registration doesn't have to be the same.
 
+Since a website will most likely download resources from the network when processing a
+`periodicsync` event, multiple `periodicsync` events can cause a lot of data consumption. Like other
+service worker events, `periodicsync` MAY be subject to the service worker execution timeout. Every
+time the browser dispatches these events, the browser will potentially be kept awake for this
+duration, which is a few minutes. This can cause battery drain. The browser implementation SHOULD
+therefore ensure an appropriate interval for firing these events. It MAY cap this interval per
+origin and across origins. It MAY also enforce limits on how often it is woken up to dispatch these
+events. To decide what an appropriate interval is, consider how often the underlying operating
+system allows scheduling repetetive tasks. Android and iOS impose a minimum interval of 15 and 10
+minutes, respectively, at the time of writing. The browser MAY also use other considerations, such
```

Comment:
"Android and iOS impose...time of writing"

This creates an implication that 15/10 minutes of minimum interval is fine. It might be worth separating that out to its own paragraph, and explaining why that may not be appropriate for PWAs, particularly due to the relation established by installing the app, as well as firing an event in a browser generally creating more pressure on the system.

Reply:
Updated. I'm not entirely sure what relation you were referring to, but I've talked about resource usage attribution here. Do we want to include a specific interval as recommendation?